android/jni/app/organicmaps/Framework.cpp

```diff
@@ -1294,6 +1295,12 @@ Java_app_organicmaps_Framework_nativeGenerateRouteAltitudeChartBits(JNIEnv * env
     totalDescent = measurement_utils::MetersToFeet(totalDescent);
   }
 
+  jni::TScopedLocalRef const totalAscentString(env,
+                             jni::ToJavaString(env, ToStringPrecision(totalAscent, 0)));
+
+  jni::TScopedLocalRef const totalDescentString(env,
+                             jni::ToJavaString(env, ToStringPrecision(totalDescent, 0)));
```

Comment:
```suggestion
  jni::TScopedLocalRef const totalAscentString(env, jni::ToJavaString(env, ToStringPrecision(totalAscent, 0)));

  jni::TScopedLocalRef const totalDescentString(env, jni::ToJavaString(env, ToStringPrecision(totalDescent, 0)));
```

Reply:
Noted! Commit will be amended accordingly...